apps/activitypub/src/routes.tsx

```
@@ -1,17 +1,17 @@
-import Inbox from '@views/Inbox';
-import Moderation from '@views/Preferences/components/Moderation';
-import Note from './views/Feed/Note';
-import Notifications from '@views/Notifications';
-import Onboarding from '@components/layout/Onboarding';
-import OnboardingStep1 from '@components/layout/Onboarding/Step1';
-import OnboardingStep2 from '@components/layout/Onboarding/Step2';
-import OnboardingStep3 from '@components/layout/Onboarding/Step3';
-import Preferences from '@views/Preferences';
-import Profile from '@views/Profile';
+import BlueskySharing from '@views/preferences/components/bluesky-sharing';
+import Error from '@components/layout/error';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)